chart/values.yaml

```diff
@@ -0,0 +1,49 @@
+CliVersion: linkerd/cli edge-19.2.5
+ControllerComponentLabel: linkerd.io/control-plane-component
+ControllerImage: gcr.io/linkerd-io/controller:edge-19.2.5
+ControllerLogLevel: info
+ControllerReplicas: 1
+ControllerUID: 2103
+CreatedByAnnotation: linkerd.io/created-by
+DestinationAPIPort: 8086
+EnableH2Upgrade: true
+EnableHA: false
+EnableTLS: false
+GlobalConfig: '{"linkerdNamespace":"linkerd","cniEnabled":false,"identityContext":null}'
+GrafanaImage: gcr.io/linkerd-io/grafana:edge-19.2.5
+GrafanaVolumeName: data
+IgnoreInboundPorts: 4190,4191
+ImagePullPolicy: IfNotPresent
+InboundAcceptKeepaliveMs: 10000
+InboundPort: 4143
+Namespace: linkerd
+NoInitContainer: false
+OutboundConnectKeepaliveMs: 10000
+OutboundPort: 4140
+ProfileSuffixes: .
+PrometheusImage: prom/prometheus:v2.7.1
+PrometheusVolumeName: data
+ProxyAutoInjectEnabled: false
+ProxyConfig: '{"proxyImage":{"imageName":"gcr.io/linkerd-io/proxy:edge-19.2.5","pullPolicy":"IfNotPresent"},"proxyInitImage":{"imageName":"gcr.io/linkerd-io/proxy-init:edge-19.2.5","pullPolicy":"IfNotPresent"},"destinationApiPort":{"port":8086},"controlPort":{"port":4190},"ignoreInboundPorts":[],"ignoreOutboundPorts":[],"inboundPort":{"port":4143},"metricsPort":{"port":4191},"outboundPort":{"port":4140},"resource":{"requestCpu":"","requestMemory":"","limitCpu":"","limitMemory":""},"proxyUid":"2102","logLevel":{"level":"warn,linkerd2_proxy=info"},"disableExternalProfiles":false}'
+ProxyContainerName: linkerd-proxy
+ProxyControlPort: 4190
+ProxyImage: gcr.io/linkerd-io/proxy:edge-19.2.5
+ProxyInitImage: gcr.io/linkerd-io/proxy-init:edge-19.2.5
+ProxyInitSpecFileName: proxy-init.yaml
+ProxyInjectAnnotation: linkerd.io/inject
+ProxyInjectDisabled: disabled
+ProxyLogLevel: warn,linkerd2_proxy=info
+ProxyMetricsPort: 4191
+ProxySpecFileName: proxy.yaml
+ProxyUID: 2102
+SingleNamespace: false
+TLSCertFileName: certificate.crt
+TLSIdentityVolumeSpecFileName: linkerd-secrets.yaml
+TLSPrivateKeyFileName: private-key.p8
+TLSSecretsVolumeName: linkerd-secrets
+TLSTrustAnchorConfigMapName: linkerd-ca-bundle
+TLSTrustAnchorFileName: trust-anchors.pem
+TLSTrustAnchorVolumeName: linkerd-trust-anchors
+TLSTrustAnchorVolumeSpecFileName: linkerd-trust-anchors.yaml
+UUID: 1e0f82c4-0b28-42b9-ba3c-c7787b8b7d58
```

Comment:
This will need to be auto-generated as it is unique per control plane.

Reply:
I still need to figure out the best way to generate this.